src/components/date_picker/super_date_picker/super_update_button.tsx

```diff
@@ -230,4 +191,57 @@ export class EuiSuperUpdateButton extends Component<
       </EuiToolTip>
     );
   }
+
+  private renderButtonContent(): ReactNode {
```

Comment:
not a change request, asking out of curiosity - why bother with `private` and a return type, vs., e.g.

```suggestion
renderButtonContent = () => {
```

Reply:
I like being strict with access modifiers and return types. In this case, we don't want anybody calling these methods from the outside, even if it's extremely unlikely for anyone to do so

Reply:
gotcha, good to know!